src/test/java/uk/ac/cam/cl/dtg/isaac/api/managers/EventBookingManagerTest.java

```diff
@@ -929,8 +930,8 @@ public void requestReservations_cancelledReservationsDoNotCountTowardsReservatio
     List<EventBookingDTO> actualResults =
         eventBookingManager.requestReservations(testCase.event, students, testCase.teacher);
     List<EventBookingDTO> expectedResults = ImmutableList.of(testCase.student1Booking);
-    assertEquals("Student 1 should get reserved despite the existing cancelled reservation", expectedResults,
-        actualResults);
+    Assertions.assertEquals(expectedResults, actualResults,
```

Comment:
You're already importing `Assertions`, so you should be able to remove it from here.